op-dispute-mon/mon/types/types.go

```diff
@@ -59,13 +59,51 @@ type EnrichedGameData struct {
 	// This ETH balance will be used to pay out any bonds required by the games
 	// that use the same DelayedWETH contract.
 	ETHCollateral *big.Int
+
+	// RollupEndpointErrors stores endpoint IDs that returned errors other than "not found" for this game.
+	RollupEndpointErrors map[string]bool
+
+	// RollupEndpointErrorCount tracks the total number of errors for this game across all endpoints.
+	RollupEndpointErrorCount int
+
+	// RollupEndpointNotFoundCount tracks the number of endpoints that returned "not found" for this game.
+	RollupEndpointNotFoundCount int
+
+	// RollupEndpointTotalCount tracks the total number of rollup endpoints attempted for this game.
+	RollupEndpointTotalCount int
+
+	// RollupEndpointSafeCount tracks the number of rollup endpoints that reported the root as safe.
+	RollupEndpointSafeCount int
+
+	// RollupEndpointUnsafeCount tracks the number of rollup endpoints that reported the root as unsafe.
+	RollupEndpointUnsafeCount int
+
+	// RollupEndpointDifferentOutputRoots tracks whether rollup endpoints returned different output roots for this game.
+	RollupEndpointDifferentOutputRoots bool
```

Comment:
There's no way to observe the value of this field. I don't see a log or a metric set when this is updated.